src/theme.ts

```diff
@@ -407,6 +407,30 @@ export const COLORS: {[name: string]: Theme} = {
     DEFAULT_RANK_SHADOW: "#6272a4",
     DEFAULT_RANK_TEXT: "#0d1117",
   },
+  github: {
```

Comment:
There is already a theme called darkhub, what's the difference?

Reply:
This is Github dark dimmed theme, and this is different from that.

Reply:
OK.  You should be renamed to identify the theme as "GitHub dark dimmed".

Reply:
Ok